build/crds/build-crds.sh

```diff
@@ -118,4 +109,7 @@ for crd in "$OLM_CATALOG_DIR/"*.yaml; do
   cat "$crd" >> "$CRDS_FILE_PATH"
 done
 
+# process final CRD file with YQ to enforce consistent indentation/formatting
+# $YQ_BIN_PATH write --inplace "$CRDS_FILE_PATH" '**'
+
```

Comment:
Added this in response to comment here: https://github.com/rook/rook/pull/7631/files#r618615480

This will enforce the YQ formatting globally on CRDs and not just specific sub-CRDs. This does also mean there is still a lot of churn in this PR, but future PRs should be saved.

Reply:
Should this be uncommented? If yq is still run on crds.yaml, seems like we wouldn't have that much churn in this PR since we were using yq previously already.

Reply:
We only used YQ previously on the CephCluster CRD file, not on the whole `crds.yaml` file or any of the other CRs.

Reply:
Also, yes it should be uncommented. I think that was just an oops from my testing.